scripts/TestHarness/testers/RavenFramework.py

```diff
@@ -98,23 +100,41 @@ def __init__(self, name, params):
     self.driver = os.path.join(RAVEN_DIR,'Driver.py')
 
   def checkRunnable(self, option):
-    missing,too_old = _missing_modules, _too_old_modules
+    missing = _missing_modules
+    too_old = _too_old_modules
+    # remove tests based on skipping criteria
+    ## required module is missing
     if len(missing) > 0:
       self.setStatus('skipped (Missing python modules: '+" ".join(missing)+
                      " PYTHONPATH="+os.environ.get("PYTHONPATH","")+')',
                      self.bucket_skip)
       return False
+    ## required module is present, but too old
     if len(too_old) > 0  and RavenUtils.checkVersions():
       self.setStatus('skipped (Old version python modules: '+" ".join(too_old)+
                      " PYTHONPATH="+os.environ.get("PYTHONPATH","")+')',
                      self.bucket_skip)
       return False
+    ## an environment varible value causes a skip
     if len(self.specs['skip_if_env']) > 0:
       env_var = self.specs['skip_if_env']
       if env_var in os.environ:
         self.setStatus('skipped (found environmental variable "'+env_var+'")',
                        self.bucket_skip)
         return False
+    ## OS
+    if len(self.specs['skip_if_OS']) > 0:
+      skip_os = [x.strip().lower() for x in self.specs['skip_if_OS'].split(',')]
+      print 'skip:',skip_os
+      # get simple-name platform (options are Linux, Windows, Darwin, or SunOS that I've seen)
+      current_os = platform.system().lower()
```

Comment:
Do we need to follow the camelBack for our scripts?

Reply:
I can change this.